docs/source/scheduler-choice.rst

```diff
@@ -0,0 +1,113 @@
+Choosing between Schedulers
+===========================
+
+Dask enables you to run your computations on a variety of schedulers that use
+different technologies, ranging from a single thread, to multiple processes, to
+a distributed cluster.  This document helps explain the differences between the
+different choices so that users can make decisions that improve performance.
+
+Briefly, the current options are as follows:
+
+*   ``dask.threaded.get``: Use multiple threads in the same process.  Good for
+    numeric code that releases the GIL (NumPy, Pandas, SKLearn, Numba) because
+    data is free to share.  The default on dask.array and dask.dataframe and
+    dask.delayed
```

Comment:
Perhaps:

"The default scheduler for ``dask.array``, ``dask.dataframe``, and ``dask.delayed``."